.github/workflows/nodejs.yml

```diff
@@ -9,12 +9,12 @@ jobs:
       postgres:
         image: postgres
         env:
-          POSTGRES_USER: alchemy
-          POSTGRES_PASSWORD: alchemy
+          POSTGRES_USER: postgres
```

Comment:
This is most likely why your tests are not working for GithubCI. This shouldn't have been changed. This script runs when you push code to Github. Please change back.